app/lib/actions/katello/capsule_content/create_or_update.rb

```diff
@@ -0,0 +1,46 @@
+module Actions
+  module Katello
+    module CapsuleContent
+      class CreateOrUpdate < ::Actions::EntryAction
+        # @param capsule_content [::Katello::CapsuleContent]
+        # @param pulp_repo [::Katello::Glue::Pulp::Repo]
```

Comment:
Do these @param comments represent the inputs to the 'plan' method?  If so, should they be updated to remove repo and add environment and content_view?